webapp/components/admin_console/save_button.jsx

```diff
@@ -22,15 +23,15 @@ export default class SaveButton extends React.Component {
     }
 
     render() {
-        const {saving, disabled, ...props} = this.props; // eslint-disable-line no-use-before-define
+        const {saving, disabled, messageId, ...props} = this.props; // eslint-disable-line no-use-before-define
 
         let contents;
         if (saving) {
             contents = (
                 <span>
                     <span className='icon fa fa-refresh icon--rotate'/>
                     <FormattedMessage
-                        id='admin.saving'
+                        id={messageId || 'admin.saving'}
```

Comment:
Instead of using this to provide a default value, you can do it above in the static defaultProps function by having it return 'admin.saving' for the messageId property